build.sbt

```diff
@@ -7,3 +57,4 @@ scalaVersion := "2.13.8"
 libraryDependencies += "org.scalameta" %% "scalameta" % "4.4.34"
 libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % Test
 libraryDependencies += "junit" % "junit" % "4.13.2"
```

Comment:
It's better to group the dependencies under the settings of a subproject. The [documentation](https://www.scala-sbt.org/1.x/docs/Multi-Project.html) might be helpful.